runtime/compiler/build/toolcfg/aix-xlc/common.mk

```diff
@@ -126,6 +131,11 @@ ifneq (,$(findstring xlclang++,$(notdir $(CXX))))
   CXX_FLAGS+=-fno-rtti
 endif
 
+ifdef I5_VERSION
+# Added i5 os specific flags
+  CX_FLAGS_EXTRA+=$(USERCFLAGS)
```

Comment:
Could you clarify why ths line is only needed for I5_VERSION? It seems to be general.

Reply:
USERCFLAGS is set by our build script to add i5 os specific flags.

Reply:
@gita-omr